proxy/ParentSelection.cc

```diff
@@ -1105,6 +1107,11 @@ EXCLUSIVE_REGRESSION_TEST(PARENTSELECTION)(RegressionTest * /* t ATS_UNUSED */,
     params->findParent(request, result, fail_threshold, retry_time); \
   } while (0)
 
+#define SET_MAX_RETRIERS(x)                                                                     \
```

Comment:
This should be an inline function, not a macro.

Reply:
@SolidWallOfCode For now, I was using the macro to be consistent with the rest of the regression tests, there are 215 regression tests all using half a dozen macro's.  I think it would be best if I create an issue and change the parent selection regression tests to use the Catch test framework instead.